tests/php/test-class-amp-auto-lightbox-disable-sanitizer.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * Class AMP_Auto_Lightbox_Disable_Sanitizer_Test.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+use AmpProject\AmpWP\Tests\Helpers\MarkupComparison;
+use AmpProject\AmpWP\Tests\TestCase;
+
+/**
+ * Tests the auto lightbox disable sanitizer class.
+ *
+ * @coversDefaultClass AMP_Auto_Lightbox_Disable_Sanitizer
+ */
+class AMP_Auto_Lightbox_Disable_Sanitizer_Test extends TestCase {
+
+	use MarkupComparison;
```

Comment:
It seems the `MarkupComparison` helper is not used in this test case. I think it can be removed.